x-pack/plugins/security_solution/server/lib/risk_engine/risk_engine_data_client.ts

```diff
@@ -57,10 +130,209 @@ export class RiskEngineDataClient {
       index,
       logger: this.options.logger,
     });
+
     this.writerCache.set(namespace, writer);
     return writer;
   }
 
+  public async getStatus({
+    savedObjectsClient,
+    namespace,
+  }: SavedObjectsClients & {
+    namespace: string;
+  }) {
+    const riskEgineStatus = await this.getCurrentStatus({ savedObjectsClient });
+    const legacyRiskEgineStatus = await this.getLegacyStatus({ namespace });
+    const lastUpdatedBy = await this.getLastUpdatedBy({ savedObjectsClient });
+    return { riskEgineStatus, legacyRiskEgineStatus, lastUpdatedBy };
+  }
+
+  public async enableRiskEngine({ savedObjectsClient, user }: UpdateConfigOpts) {
+    // code to run task
+
+    return this.udpateSavedObjectAttribute({
+      savedObjectsClient,
+      user,
+      attributes: {
+        enable: true,
+      },
+    });
+  }
+
+  public async disableRiskEngine({ savedObjectsClient, user }: UpdateConfigOpts) {
+    // code to stop task
+
+    return this.udpateSavedObjectAttribute({
+      savedObjectsClient,
+      user,
+      attributes: {
+        enable: false,
+      },
+    });
+  }
+
+  private async udpateSavedObjectAttribute({
+    savedObjectsClient,
+    attributes,
+    user,
+  }: UpdateConfigOpts & {
+    attributes: {
+      enable: boolean;
+    };
+  }) {
+    const savedObjectConfiguration = await this.getConfigurationSavedObject({
+      savedObjectsClient,
+    });
+
+    if (!savedObjectConfiguration) {
+      throw new Error('There no saved object configuration for risk engine');
+    }
+
+    const result = await savedObjectsClient.update(
+      riskEngineConfigurationTypeName,
+      savedObjectConfiguration.id,
+      {
+        ...attributes,
+        last_updated_by: user?.username ?? '',
```

Comment:
I think saved_object itself has this date for the last update.

Probably when we have a solution for a chronological history of actions, then we should also save what type of actions was made. But for now, probably it should be enough.

Reply:
++ @rylnd It would be ideal to display enablement/ disablement activity separately from subsequent risk configuration changes. 

Typically, the enablement is a Detection Engineer (DE) persona and Risk Configuration can (Analyst and DE). Although we don't have the RBAC granularity for enforcing, displaying this explicitly in the audit log will reduce user confusion.

Does this answer the question?